buildpacks/builder.go

```diff
@@ -28,14 +28,17 @@ var runtime2pack = map[string]string{
 	"go":      "quay.io/boson/faas-go-builder",
 }
 
-func (builder *Builder) Build(name, runtime, path string) (image string, err error) {
+func (builder *Builder) Build(path string) (image string, err error) {
```

Comment:
unused named param `image`

Reply:
also `err` is returned explicitly

Reply:
I didn't quite understand how those return statements were working/compiling. This clears it up. TY.